src/registry/LocalRegistry.ts

```diff
@@ -89,6 +91,65 @@ export class LocalRegistry extends BaseRegistry implements IRegistry {
     return addresses[chainName];
   }
 
+  addChains(
+    chains: Array<{ chainName: ChainName; metadata?: ChainMetadata; addresses?: ChainAddresses }>,
+  ): void {
+    const currentChains = this.listRegistryContent().chains;
+    for (const chain of chains) {
+      if (currentChains[chain.chainName])
+        throw new Error(`Chain ${chain.chainName} already exists in registry`);
+    }
```

Comment:
should we throw or merge here?

Reply:
IMO people should use `updateChain` if they want add-or-merge behavior